pkg/combustion/cleanup.go

```diff
@@ -0,0 +1,37 @@
+package combustion
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/suse-edge/edge-image-builder/pkg/fileio"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+	"github.com/suse-edge/edge-image-builder/pkg/log"
+	"go.uber.org/zap"
+)
+
+const (
+	cleanupScriptName    = "cleanup-combustion.sh"
+	cleanupComponentName = "cleanup"
+)
+
+//go:embed templates/cleanup-combustion.sh
+var cleanupScript string
+
+func configureCleanup(ctx *image.Context) ([]string, error) {
```

Comment:
I'm wondering if we should still treat this as a component. Now that it's out of the loop, it's just a file that we write on the filesystem.

Reply:
I wondered this too but didn't have strong opinions in either direction, for whatever it's worth.